src/core/a-node.js

```diff
@@ -39,7 +39,7 @@ class ANode extends HTMLElement {
       document.addEventListener('DOMContentLoaded', this.connectedCallback.bind(this));
       return;
     }
-    ANode.prototype.doConnectedCallback.call(this);
+    this.doConnectedCallback();
```

Comment:
For a-scene, a-assets, a-mixin, we can use `this.doConnectedCallback();` because we don't inherit from those.
But for a-node and a-entity, you can't use it, because it will call the `doConnectedCallback` from the derived class, we don't want that here.